### PR TITLE
Updated Activity Feed with the Event Class Implementation

### PR DESCRIPTION
**Pull request by lealgamb**
I updated the appropriate servlets that stored information to the datastore by storing to the EventStore. Then I updated how the Activity Feed page is rendered. To see these changes locally, you'll have to run the clean command.

I will work on updating the tests.

### CODE REVIEW

**Comment by lealgamb** [comment]
I had to update `addEvent()` in `EventStore` to the following: `events.add(0, event)` to store the list from most recent to oldest. 

I also have a question: When the "About Me" field is updated in `ProfileServlet`, do I also have to update the `Event` object? I was unsure how to do this.

**Comment by jasminechau2** [comment]
If "About Me" is stored within the user object, when "About Me" is not null / an empty string. You can find the name of the user stored in the event store and then insert the update "About Me" into the information list. I don't know if this works, but maybe something along this idea could help you.

File: src/main/java/codeu/model/store/basic/EventStore.java [hunk 1]
```diff
@@ -59,7 +59,14 @@ private EventStore(PersistentStorageAgent persistentStorageAgent) {
    * to add a new event, not to update an existing event.
    */
   public void addEvent(Event event) {
-    events.add(event);
+    events.add(0, event);
```
[the remark this answers is not in the document]
Comment (yebrahim) [reply src/main/java/codeu/model/store/basic/EventStore.java]: Why do we prepend? Why not append to the end of the list?

File: src/main/java/codeu/model/store/basic/EventStore.java [hunk 2]
```diff
@@ -59,7 +59,14 @@ private EventStore(PersistentStorageAgent persistentStorageAgent) {
...
+    persistentStorageAgent.writeThrough(event);
+  }
+
+  /**
+   * Update an existing Event.
+   */
+  public void updateEvent(Event event) {
```
[the remark this answers is not in the document]
Comment (yebrahim) [reply src/main/java/codeu/model/store/basic/EventStore.java]: +1. Let's start simple by counting any activity on the app as a separate event.

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 3]
```diff
@@ -116,6 +116,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     //set user's about me and update it in dataStore
     subject.setAboutMe(cleanedAboutMe);
     userStore.updateUser(subject);
+    // TODO figure out how to update the event
```
[the remark this answers is not in the document]
Comment (lealgamb) [reply src/main/java/codeu/controller/ProfileServlet.java]: That's a great idea.

File: src/main/java/codeu/model/store/basic/EventStore.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (lealgamb) [reply src/main/java/codeu/model/store/basic/EventStore.java]: It was to have the list from most recent to oldest, but I realize now that I can append and then have a for loop from `[size - 1, 0]`.

File: src/main/java/codeu/model/store/basic/EventStore.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (lealgamb) [reply src/main/java/codeu/model/store/basic/EventStore.java]: Okay, I'll remove this method.

File: src/main/java/codeu/model/store/basic/EventStore.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (yebrahim) [reply src/main/java/codeu/model/store/basic/EventStore.java]: Yea, looping backward is almost as common as forward, though it doesn't readily come to mind. It's so common that lists are typically implemented as a doubly linked list to allow for it, as is the case for C++ STL. Not sure about Java, but probably the same?

**Pull request merged**